.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "type": "rdbg",
+      "name": "Attach with rdbg",
+      "request": "attach",
+      "rdbgPath": "${workspaceFolder}/Library/Homebrew/vendor/portable-ruby/current/lib/ruby/gems/3.3.0/gems/debug-1.9.1/exe/rdbg",
```

Comment:
I wonder if it's worth having our own wrapper script/command for this that avoids needing to hardcode into VSCode config?

Reply:
Agreed, but I think I'll do this in a follow-up PR since I have some thoughts on how to do this best in a way to allow us to launch debug sessions from VS Code as well